pkg/local_workflows/config_workflow.go

```diff
@@ -35,6 +44,27 @@ func InitConfigWorkflow(engine workflow.Engine) error {
 	return err
 }
 
+type SnykRegion struct {
+	alias string
+	url   string
+}
```

Comment:
question: what's the reason for changing from a map to a struct?

Reply:
How would the map help with that?

Reply:
nvm, I think I convinced myself out of it. It may actually complicate things if the map has to contain both the alias and url as keys to be searchable.